string/reverse_string.py

```diff
@@ -19,7 +19,15 @@ def iterative(s):
 print(iterative(s))
 
 def pythonic(s):
-    r = list(reversed(s))
-    return "".join(r)
+	r = list(reversed(s))
```

Comment:
Would recommend to skip this function, as it does not illustrate the algorithm behind reversing a string.